cmd/build.go

```diff
@@ -603,3 +607,19 @@ func mapLinkedVars(mapping map[string]interface{}, manifestPath string, config *
 
 	return nil
 }
+
+// HumanizeKustomizeError takes errors returned from GenerateKustomizeResult and returns them in a human friendly string
+func HumanizeKustomizeError(err error) string {
+	if paramsError, ok := err.(*manifest.ParamsError); ok {
+		switch paramsError.ErrorType {
+		case "missing":
+			return fmt.Sprintf("%s is missing in your params.yaml", paramsError.Key)
+		case "blank":
```

Comment:
Or maybe these `ErrorType`s should be `namespaceBlank` and `namespaceReserved`?

Reply:
@rushtehrani That might be best. Although, it is possible other params could be missing or blank when they should not be - the messages would be the same.